fapolicy_analyzer/ui/main_window.py

```diff
@@ -303,3 +340,76 @@ def on_trustDbMenu_activate(self, menuitem, *args):
     def on_deployChanges_clicked(self, *args):
         with DeployChangesetsOp(self.window) as op:
             op.run(self._changesets)
+
+# ###################### fapolicyd interfacing ##########################
+    def on_fapdStartMenu_activate(self, menuitem, data=None):
+        logging.debug("on_fapdStartMenu_activate() invoked.")
+        if (self._fapd_status != ServiceStatus.UNKNOWN) and (self._fapd_lock.acquire()):
+            self._fapd_ref.start()
+            self._fapd_lock.release()
+
+    def on_fapdStopMenu_activate(self, menuitem, data=None):
+        logging.debug("on_fapdStopMenu_activate() invoked.")
+        if (self._fapd_status != ServiceStatus.UNKNOWN) and (self._fapd_lock.acquire()):
+            self._fapd_ref.stop()
+            self._fapd_lock.release()
+
+    def _enable_fapd_menu_items(self, status):
+        if self._fapdControlPermitted and (status != ServiceStatus.UNKNOWN):
+            self._fapdStartMenuItem.set_sensitive(not status)
+            self._fapdStopMenuItem.set_sensitive(status)
+        else:
+            self._fapdStartMenuItem.set_sensitive(False)
+            self._fapdStopMenuItem.set_sensitive(False)
+
+    def _update_fapd_status(self, status: ServiceStatus):
+        logging.debug(f"__update_fapd_status({status})")
+
+        # Enable/Disable fapd menu items
+        self._enable_fapd_menu_items(status)
+        if status is True:
+            self.fapdStatusLight.set_from_stock(stock_id="gtk-yes", size=4)
+        elif status is False:
+            self.fapdStatusLight.set_from_stock(stock_id="gtk-no", size=4)
```

Comment:
For consistency these comparisons should probably be `status == ServiceStatus.TRUE` and `status == ServiceStatus.FALSE` the same as all your comparisons with `ServiceStatus.UNKNOWN` in other places.

Reply:
Done